### ISSUE TITLE
SIP bridge is not being activated for a new meeting before the first browser-based audio user joins

### ISSUE THREAD

**Issue by steffen-moser** [bigbluebutton/bigbluebutton#10083]
**Describe the bug**
The audio bridge to the VoIP system is opened not before the first web-based logged-in user activates his or her web-based (WebRTC or Flash) audio. 

This means: Users, who do want to handle their voice via telephone (e.g. because their headset is broken/unavailable or due to audio distortion caused by a congested internet link) and want to follow the presentation via web, typically get fooled if such a user is the first one who joins the conference. They start BBB, don't activate web-based audio and try to call the telephone number and enter the (static or randomly generated) PIN when asked. If there isn't any other user in the meeting who already activated his/her web-based audio, they will hear the busy signal (hangup).

For this reason, I would highly appreciate if BBB registered an (internal dummy) audio session to the voice subsystem immediately after the first user has opened the BBB session. I think this would be a more intuitive behavior of the system.

**To Reproduce**
 - Open a BBB new (!) meeting
 - Click on "Join using your phone"
 - Dial the given phone number
 - Wait for being asked for the PIN
 - Enter the PIN
 - You'll hear a busy signal (due to FreeSwitch hanging up the line).

**Expected behavior**
Also the first user should be connected to the meeting room via phone.

**Actual behavior**
The voice bridge for a given meeting will be only activated after at least one user connected his/her audio using the computer. In the most cases, the instructor uses her or his computer and is the first one who joins a room. But in rare cases, instructors had trouble to use audio (see also #9295) and wanted to join via phone. This didn't work as a fallback then.

**Comment by lonesomewalker** [comment]
This is a duplicate.
We already discussed this here several times.
In general ONE person has to join with audio, otherwise this also makes no sense.

**Comment by steffen-moser** [comment]
Sorry, but I don't get why this should make no sense... 

There are real cases our support team sees several times a week: We do have instructors behind strictly configured firewalls, behind rare SQUID proxy servers or instructors with just poor internet connectivity at their homes (in Germany, there are still villages without functional broadband connectivity). They are required to use their phones in order to speak to their listeners. 

Teachers are the first ones who typically join a session. BBB offers to use the phone, it also does offer this to the first user. BBB's current behavior (offering to use the phone and hanging up the line) is neither intuitive nor unavoidable. I don't see why it is difficult to let BBB open an internal muted connection to FreeSwitch and to dump the incoming packets. It could also just make Kurento connect to FreeSwitch (this is what happens when the first user presses "listen-only") automatically after opening the room. I am quite sure that a very simple measure could solve the problem and make this great tool more userfriendly. 

Currently, if an instructor comes to a place where joining audio does not work, he or she tries the phone, hears the busy signal after typing the PIN, and exits the session. The session does not take place, the students are upset and in the best case, the affected instructors open a ticket a the support desk where they are being told to open "listen-only" and switch off the speakers or to wait until the first students join. In the not so good cases, they just grouse to their colleagues about the non-working tools we provide. We especially want to prevent the latter.

**Comment by mtelgkamp** [comment]
Maybe the workaround of https://github.com/bigbluebutton/bigbluebutton/issues/9161#issuecomment-617601390 works for you?

**Comment by lonesomewalker** [comment]
I did not say it makes no sense.
I wrote we already had this issue here.
As @mtelgkamp wrote the issue, there you also commented.
So, instead of complaining again, feel free to join the developers :-)
Sure, not easy, but in general how the situation is done right now works for most of the users.
And it should be possible to at least have one user join by headset...?

**Comment by steffen-moser** [comment]
Honestly, I actually forgot that I had already complained about this in May. I am very sorry about this! 

I've just seen my old posting when @mtelgkamp mentioned the issue #9161 and a colleague of mine pointed it out at the same time... :-| As an excuse, I could probably only say that my first complaint was in the midst of the Coronavirus lockdown where we worked day and night in order to make e-learning function. :-) When I now start thinking about it, I am even quite sure that I wrote about this behavior also about 10 months ago in the bbb-developer group, but it seems that I didn't state it clearly enough.

And yes, I took this problem on the list of the topics a student of mine is having a look at. Maybe we can solve it by letting Kurento open an automatic dummy connection to FreeSwitch as soon as a meeting is started - even before anyone joins by headset or by a microphone.

Of course, the teachers who have already been hit by this problem know about it now and join via headset or wait for a student joining with audio. The problem is: The users who open the support tickets are typically the ones who are unable to cope with the problem when it occurs for the first time. They panic, the session won't take place, and afterward, they complain about the tool. We do also seem to have very few users who sit behind a strict or misconfigured firewall or proxy solution, they just want to show some slides and talk. They cannot establish any WebRTC connection at all while showing slides and speaking via phone would at least deliver a minimum level of quality. 

So as it is clearly a duplicate of #9161, I am going to close this. Sorry for any inconvenience and trouble.
